src/components/icon/icon.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { library } from '@fortawesome/fontawesome-svg-core';
+
+import { fas }  from '@fortawesome/free-solid-svg-icons';
+import { fab } from '@fortawesome/free-brands-svg-icons';
+// Added for future compatibility with websites that subscribe to regular icons 
+import { far }  from '@fortawesome/free-regular-svg-icons';
+
+library.add(fas, fab, far)
+
+type IconTypes = {
```

Comment:
I am wondering if we should add some of the other propTypes that FontAwesomeIcon takes as well. At least the most important ones. I would vote for adding _color_ and _size_ at least. In addition to changing _icon_ to _string|string[]_, Thoughts? 🤔

Reply:
Thanks for the proposition previous month, and apologies for the slow reply. Sounds like a good idea to add to the Icon component. Started working on adding the propTypes to Icon.